components/site/src/lib.rs

```diff
@@ -901,6 +901,24 @@ impl Site {
             }
         }
 
+        if section.meta.generate_feed {
+            let library = &self.library.read().unwrap();
+            let pages = section
+                .pages
+                .iter()
+                .map(|k| library.get_page_by_key(*k))
+                .collect();
+            self.render_feed(
+                pages,
+                Some(&PathBuf::from(&section.path[1..])),
+                &section.lang,
+                |mut context: Context| {
+                    context.insert("section", &section.to_serialized(library));
+                    context
+                },
+            )?;
+        }
```

Comment:
looks like there are unrelated changes in the commits

Reply:
probably my mistake while pulling `next` so the PR wouldn't have merge conflicts, I'll remove it